### ISSUE TITLE
[A11Y] Menu item that shows a submenu is a link instead of a button

### ISSUE THREAD

**Issue by israelcefrin** [pkp/immersion#142]
## Issue Description
When a menu item has subitems, it remains as  link element `<a href=""> </a>` and an ARIA role=button is added to it. It keeps an URL that doesn't work anymore. 

## How to reproduce / Current behavior
1) Access a [journal that uses Immersion theme](https://ojs34.demo.publicknowledgeproject.org/index.php/immersion)
2) Click on the menu item that shows an arrow (subitems)
3) It will open a submenu, but the element still has the URL.

## What application are you using?
 OJS version 3.4

## Stack
- Device:  Laptop
- OS:  MacOS Sequoia 
- Browser: Firefox

## Testing method
- Method: Manual, Keyboard

## Action
It is possible to fix it changing the HTML markup from `LINK` to `button` for elements in the menu that has subitems.

## Additional information

<img width="1633" height="678" alt="Image" src="https://github.com/user-attachments/assets/5592a051-c0d2-4ca0-a128-bad50e40d7c1" />

**Comment by israelcefrin** [comment]
Hi @kaitlinnewson , just pinging to let you know that PR is ready for review and testing.
